changedetectionio/static/js/visual-selector.js

```diff
@@ -186,22 +187,26 @@ $(document).ready(function () {
                 if (e.offsetY > sel.top * y_scale && e.offsetY < sel.top * y_scale + sel.height * y_scale
                     &&
                     e.offsetX > sel.left * y_scale && e.offsetX < sel.left * y_scale + sel.width * y_scale
-
+                    &&
+                    sel.width * sel.height < minSquare
                 ) {
 
                     // FOUND ONE
-                    set_current_selected_text(sel.xpath);
-                    ctx.strokeRect(sel.left * x_scale, sel.top * y_scale, sel.width * x_scale, sel.height * y_scale);
-                    ctx.fillRect(sel.left * x_scale, sel.top * y_scale, sel.width * x_scale, sel.height * y_scale);
+                    found = sel;
 
                     // no need to keep digging
                     // @todo or, O to go out/up, I to go in
                     // or double click to go up/out the selector?
                     current_selected_i = i;
-                    found += 1;
-                    break;
+                    minSquare = sel.width * sel.height;
                 }
             }
+            
+            if(found != null) {
```

Comment:
```suggestion
            if(found) {
```

?

Reply:
Initially `found` is set to `null`, so I prefer to check if its initial value was never changed (undefined, etc)